src/vma/dev/net_device_val.cpp

```diff
@@ -94,6 +94,15 @@ net_device_val::~net_device_val()
 		delete m_p_L2_addr;
 		m_p_L2_addr = NULL;
 	}
+
+	for (slave_data_vector_t::iterator it = m_slaves.begin() ; it != m_slaves.end(); ++it) {
+		slave_data_t* sd = *it;
+		if (sd->if_name) {
+			free(sd->if_name);
+		}
+		delete sd->p_L2_addr;
```

Comment:
@serglelv please follow convention in this file and assign NULL to deleted element
```
delete sd->p_L2_addr;
sd->p_L2_addr = NULL;
```